include/rocksdb/wide_columns.h

```diff
@@ -218,4 +220,44 @@ inline bool operator!=(const PinnableWideColumns& lhs,
   return !(lhs == rhs);
 }
 
+// PinnableWideColumns bundled with ColumnFamilyHandle and Status for the query.
+// Used for the results of wide-column queries grouped by Column Families
+class PinnableWideColumnsBundle {
```

Comment:
Thought about the naming a bit more; since we've been talking about attribute groups, it would probably make sense to reflect that in the API too. So how about calling this class `PinnableWideAttributeGroup` and the collection `PinnableWideAttributeGroups` (plural)?

Reply:
How about `PinnableAttributeGroup`? We are defining AttributeGroup as what wide-column entities can be split into. WideAttributeGroup sounds little awkward to me.